features/GettingStarted.md

```diff
@@ -29,15 +29,15 @@ the files in the `spec` directory to RSpec.
 
 ### Run migrations
 
-    $ rake db:migrate && rake db:test:prepare
+    $ rails db:migrate && rails db:test:prepare
 
 ### Run RSpec
 
     $ rake spec
```

Comment:
Never using this, not sure if it can/should be `rails spec` now or `rake` is preferrable just like with running other common tasks.

Reply:
Is rails available in 5?

Reply:
It is https://github.com/rails/rails/blob/5-0-stable/guides/source/command_line.md#binrails